server/routes/one-time-code-auth/OneTimeCodeEmailSentController.ts

```diff
@@ -1,12 +1,13 @@
 import { Request, Response } from 'express'
+import OneTimeCodeEmailSentView from './OneTimeCodeEmailSentView'
 
 export default class OneTimeCodeEmailSentController {
   async getOneTimeCodeEmailSentView(req: Request, res: Response): Promise<void> {
     if (!req.session.requestOneTimeCodeForm?.email) {
       return res.redirect('request-code')
     }
 
-    const emailSentTo = req.session.requestOneTimeCodeForm.email
-    return res.render('pages/one-time-code-auth/emailSent', { emailSentTo })
+    const view = new OneTimeCodeEmailSentView(req.session.requestOneTimeCodeForm.email, req.flash('errors'))
```

Comment:
We need a proper view object so that flash errors can be rendered to the screen if there are any